backend/controllers/user.controller.ts

```diff
@@ -1 +1,50 @@
-// placeholder for now
\ No newline at end of file
+import { Request, Response } from 'express';
+import HttpError from '../utils/error';
+import StatusCode from '../utils/StatusCode'
+import User, { IUser } from '../models/user.model';
+
+export const getAll = async (_req: Request, res: Response) => {
+  try {
+    const users = await User.find();
+    if (users.length === 0) {
+      throw new HttpError('Not found any users', StatusCode.NO_CONTENT);
```

Comment:
I think we can use here res.status instead of throwing an error. Could you check it?
```suggestion
      res.status(StatusCode.NO_CONTENT).json({ message: 'Not found any users' });
```

Reply:
Do You want me to create some enum with statuses ?